app/robots/iCubGenova09/virtualizerConfig.ini

```diff
@@ -2,23 +2,23 @@ name            virtualizer
 period          0.05
 
 # Minimum angular error to trigger the rotation of the robot (value in radians)
-angle_deadzone        0.35
+angle_deadzone        0.4
 
 #Minimum walking velocity to send a reference to the robot (value in meters per second)
-speed_deadzone        0.1
+speed_deadzone        0.5
 
 # RPC options
 playerOrientationPort_name    /playerOrientation:o
 robotOrientationPort_name     /robotOrientation:i
-rpcWalkingPort_name           /walkingRpc
-scale_X                       3.0
+rpcWalkingPort_name           /goal:o
```

Comment:
Please update the `rpcWalkingPort_name` with some name like `robotGoalPort_name` or sth similar.

Reply:
Please update the port name in all the xml files.

Reply:
Also, please also put the same port name for all the robot in the `.ini` files.
Thanks :)